{
	"alias": "FEDORA-2019-768ed4fddc",
	"bugs": [
		{
			"bug_id": 1687688,
			"feedback": [
				{
					"bug_id": 1687688,
					"comment": {
						"id": 909892,
						"testcase_feedback": [
							{
								"comment_id": 909892,
								"testcase": {
									"id": 371,
									"name": "QA:Testcase crontabs"
								},
								"testcase_id": 371
							}
						],
						"text": "Hmm, my cron jobs did not run after upgrade, so with this being pushed to stable, others may get stung too...",
						"timestamp": "2019-03-17 20:20:04",
						"update_id": 133860,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 909892
				},
				{
					"bug_id": 1687688,
					"comment": {
						"id": 909908,
						"karma": -1,
						"karma_critpath": -1,
						"testcase_feedback": [
							{
								"comment_id": 909908,
								"testcase": {
									"id": 371,
									"name": "QA:Testcase crontabs"
								},
								"testcase_id": 371
							}
						],
						"text": "I'm going to -1 this after checking the logs. User crontabs were ignored after upgrade, so maintainer should have a look before this hits stable.",
						"timestamp": "2019-03-17 20:52:20",
						"update_id": 133860,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 909908
				},
				{
					"bug_id": 1687688,
					"comment": {
						"id": 911048,
						"karma": 1,
						"karma_critpath": 1,
						"testcase_feedback": [
							{
								"comment_id": 911048,
								"testcase": {
									"id": 371,
									"name": "QA:Testcase crontabs"
								},
								"testcase_id": 371
							}
						],
						"text": "Works here.",
						"timestamp": "2019-03-19 11:26:47",
						"update_id": 134052,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 911048
				}
			],
			"parent": true,
			"security": true,
			"title": "CVE-2019-9704 vixie-cron: calloc return value resulting in remote dos",
			"bugzilla": {
				"bug_id": "1687688",
				"alias": "CVE-2019-9704",
				"creation_ts": "2019-03-12 07:20:54 +0000",
				"short_desc": "CVE-2019-9704 vixie-cron: calloc return value resulting in remote dos",
				"delta_ts": "2020-04-22 04:32:22 +0000",
				"bug_status": "CLOSED",
				"resolution": "WONTFIX",
				"keywords": "Security",
				"priority": "low",
				"bug_severity": "low",
				"depends_on": [
					"1699088",
					"1699089"
				],
				"blocked": [
					{
						"bug_id": "1696266",
						"error": "NotPermitted"
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "12534220",
						"comment_count": "0",
						"who": {
							"text": "darunesh",
							"name": "Dhananjay Arunesh"
						},
						"bug_when": "2019-03-12 07:20:54 +0000",
						"thetext": "Vixie Cron before the 3.0pl1-133 Debian package allows local users to cause a denial of service (daemon crash) via a large crontab file because the calloc return value is not checked.\n\nUpstream commit:\nhttps://salsa.debian.org/debian/cron/commit/f2525567"
					},
					{
						"isprivate": "0",
						"commentid": "12548964",
						"comment_count": "1",
						"who": {
							"text": "tmraz",
							"name": "Tomas Mraz"
						},
						"bug_when": "2019-03-15 13:54:33 +0000",
						"thetext": "In cronie handled in cronie-1.5.3 release."
					},
					{
						"isprivate": "0",
						"commentid": "12629157",
						"comment_count": "3",
						"who": {
							"text": "sgayou",
							"name": "Scott Gayou"
						},
						"bug_when": "2019-04-10 18:53:57 +0000",
						"thetext": "Also fixed upstream in cronie here:\nhttps://github.com/cronie-crond/cronie/commit/a6576769f01325303b11edc3e0cfb05ef382ce56"
					},
					{
						"isprivate": "0",
						"commentid": "12648566",
						"comment_count": "5",
						"who": {
							"text": "sgayou",
							"name": "Scott Gayou"
						},
						"bug_when": "2019-04-17 18:15:56 +0000",
						"thetext": "Upstream Cronie fix is in release 1.5.3."
					},
					{
						"isprivate": "0",
						"commentid": "13825158",
						"comment_count": "8",
						"who": {
							"text": "prodsec-dev",
							"name": "Product Security DevOps Team"
						},
						"bug_when": "2020-04-22 04:32:22 +0000",
						"thetext": "This bug is now closed. Further updates for individual products will be reflected on the CVE page(s):\n\nhttps://access.redhat.com/security/cve/cve-2019-9704"
					}
				]
			}
		},
		{
			"bug_id": 1687694,
			"feedback": [
				{
					"bug_id": 1687694,
					"comment": {
						"id": 909892,
						"testcase_feedback": [
							{
								"comment_id": 909892,
								"testcase": {
									"id": 371,
									"name": "QA:Testcase crontabs"
								},
								"testcase_id": 371
							}
						],
						"text": "Hmm, my cron jobs did not run after upgrade, so with this being pushed to stable, others may get stung too...",
						"timestamp": "2019-03-17 20:20:04",
						"update_id": 133860,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 909892
				},
				{
					"bug_id": 1687694,
					"comment": {
						"id": 909908,
						"karma": -1,
						"karma_critpath": -1,
						"testcase_feedback": [
							{
								"comment_id": 909908,
								"testcase": {
									"id": 371,
									"name": "QA:Testcase crontabs"
								},
								"testcase_id": 371
							}
						],
						"text": "I'm going to -1 this after checking the logs. User crontabs were ignored after upgrade, so maintainer should have a look before this hits stable.",
						"timestamp": "2019-03-17 20:52:20",
						"update_id": 133860,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 909908
				},
				{
					"bug_id": 1687694,
					"comment": {
						"id": 911048,
						"karma": 1,
						"karma_critpath": 1,
						"testcase_feedback": [
							{
								"comment_id": 911048,
								"testcase": {
									"id": 371,
									"name": "QA:Testcase crontabs"
								},
								"testcase_id": 371
							}
						],
						"text": "Works here.",
						"timestamp": "2019-03-19 11:26:47",
						"update_id": 134052,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 911048
				}
			],
			"parent": true,
			"security": true,
			"title": "CVE-2019-9705 vixie-cron: dos(memory consumption) via a large crontab file",
			"bugzilla": {
				"bug_id": "1687694",
				"alias": "CVE-2019-9705",
				"creation_ts": "2019-03-12 07:30:33 +0000",
				"short_desc": "CVE-2019-9705 vixie-cron: memory consumption DoS via a large crontab file",
				"delta_ts": "2021-10-27 03:26:58 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "Security",
				"priority": "medium",
				"bug_severity": "medium",
				"depends_on": [
					"1711356",
					"1711367",
					"1711368"
				],
				"blocked": [
					{
						"bug_id": "1687709",
						"error": "NotPermitted"
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "12534244",
						"comment_count": "0",
						"who": {
							"text": "darunesh",
							"name": "Dhananjay Arunesh"
						},
						"bug_when": "2019-03-12 07:30:33 +0000",
						"thetext": "Vixie Cron before the 3.0pl1-133 Debian package allows local users to cause a denial of service (memory consumption) via a large crontab file because an unlimited number of lines is accepted.\n\nUpstream commit:\nhttps://salsa.debian.org/debian/cron/commit/26814a26"
					},
					{
						"isprivate": "0",
						"commentid": "12548965",
						"comment_count": "1",
						"who": {
							"text": "tmraz",
							"name": "Tomas Mraz"
						},
						"bug_when": "2019-03-15 13:54:49 +0000",
						"thetext": "In cronie handled in 1.5.3 release."
					},
					{
						"isprivate": "0",
						"commentid": "12748316",
						"comment_count": "3",
						"who": {
							"text": "rschiron",
							"name": "Riccardo Schirone"
						},
						"bug_when": "2019-05-17 15:06:39 +0000",
						"thetext": "Created cronie tracking bugs for this issue:\n\nAffects: fedora-all [bug 1711356]"
					},
					{
						"isprivate": "0",
						"commentid": "12900013",
						"comment_count": "5",
						"who": {
							"text": "mplch",
							"name": "Marcel Plch"
						},
						"bug_when": "2019-06-26 07:51:54 +0000",
						"thetext": "There seems to be a use case for *not* limiting the amount of cronjobs.\n\nAniket, may I ask you to explain the use case as you explained to me via e-mail?\n\nThank you."
					},
					{
						"isprivate": "0",
						"commentid": "12900089",
						"comment_count": "6",
						"who": {
							"text": "tmraz",
							"name": "Tomas Mraz"
						},
						"bug_when": "2019-06-26 08:16:22 +0000",
						"thetext": "There always should be a limit. The only question is how high it should be."
					},
					{
						"isprivate": "0",
						"commentid": "12900135",
						"comment_count": "7",
						"who": {
							"text": "mplch",
							"name": "Marcel Plch"
						},
						"bug_when": "2019-06-26 08:21:06 +0000",
						"thetext": "Maybe it would be wise to set the limit dynamically based on system resources? Or at least load it from some config file?\nLarge mainframes should be expected to run more jobs than a personal workstation."
					},
					{
						"isprivate": "0",
						"commentid": "12900199",
						"comment_count": "8",
						"who": {
							"text": "tmraz",
							"name": "Tomas Mraz"
						},
						"bug_when": "2019-06-26 08:34:12 +0000",
						"thetext": "I do not think we want to introduce a config file just for this or complicate it by system resources estimation. At most I can see adding a new option but that still does not make much sense to me. The limit is per-crontab and affects only the user's crontab so what would be the maximum needed if the current limit of 1000 is not sufficient. 10000? 50000? I do not think anything higher is appropriate at all."
					},
					{
						"isprivate": "0",
						"commentid": "12900216",
						"comment_count": "9",
						"who": {
							"text": "mplch",
							"name": "Marcel Plch"
						},
						"bug_when": "2019-06-26 08:38:17 +0000",
						"thetext": "Let's wait for the information from Aniket and decide according to that, then."
					},
					{
						"isprivate": "0",
						"commentid": "12902499",
						"comment_count": "10",
						"who": {
							"text": "abhavsar",
							"name": "Aniket Bhavsar"
						},
						"bug_when": "2019-06-26 19:05:51 +0000",
						"thetext": "Below is the use case as per customer:\n\n~~~~\nOur applications were running on a Solaris platform. And from time to time, the limit of 100 jobs were reached (c queue max run limit reached). In that case, after a few minutes, the job in queue is dropped.\nCurrently, we don't have any issues with these applications on a Linux 7.3 platform because we already have reduced the number of jobs and the frequency of execution. But because of business, the number of jobs is increasing.\nSo we are wondering about the behaviour of Linux system...\n~~~~"
					},
					{
						"isprivate": "0",
						"commentid": "12903467",
						"comment_count": "11",
						"who": {
							"text": "tmraz",
							"name": "Tomas Mraz"
						},
						"bug_when": "2019-06-27 06:20:58 +0000",
						"thetext": "Is this really related to user cron job table limit? The current upstream limit is 1000 different user cron jobs (as created by crontab command) for a single user so that is quite far from 100.\n\nHowever the Solaris case mentioned above does not really talk about cron jobs."
					},
					{
						"isprivate": "0",
						"commentid": "12912286",
						"comment_count": "12",
						"who": {
							"text": "abhavsar",
							"name": "Aniket Bhavsar"
						},
						"bug_when": "2019-06-28 19:39:11 +0000",
						"thetext": "It more like 'How many simultaneous jobs at the time the daemon crond can execute?'."
					},
					{
						"isprivate": "0",
						"commentid": "12914758",
						"comment_count": "13",
						"who": {
							"text": "tmraz",
							"name": "Tomas Mraz"
						},
						"bug_when": "2019-07-01 07:46:29 +0000",
						"thetext": "There is no artificial limit for that."
					},
					{
						"isprivate": "0",
						"commentid": "12929087",
						"comment_count": "14",
						"who": {
							"text": "mplch",
							"name": "Marcel Plch"
						},
						"bug_when": "2019-07-03 14:44:07 +0000",
						"thetext": "Aniket, are you able to inform the customer that we plan to fix this CVE by setting the upstream limit of 1000 jobs?\nAlso note that this limit is set per-user and does not apply to root jobs.\n\nPlease, also ask for feedback whether this is alright for the customer. If they move their crontab configuration to a root's crontab, they shouldn't have to worry about the limit, the question is if this approach is feasible for them.\n\nThank you."
					},
					{
						"isprivate": "0",
						"commentid": "13063531",
						"comment_count": "17",
						"who": {
							"text": "abhavsar",
							"name": "Aniket Bhavsar"
						},
						"bug_when": "2019-08-14 04:15:06 +0000",
						"thetext": "Apology for the dealy. There was no response from the customer the case is closed now."
					}
				]
			}
		}
	],
	"builds": [
		{
			"epoch": 0,
			"nvr": "cronie-1.5.3-1.fc29",
			"release_id": 23,
			"signed": true,
			"type": "rpm",
			"package": {
				"aarch64": [
					{
						"name": "cronie-debugsource",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "cronie-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "cronie-noanacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "cronie-anacron-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "cronie",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "cronie-anacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "aarch64"
					}
				],
				"armv7hl": [
					{
						"name": "cronie-anacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "cronie-debugsource",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "cronie-noanacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "cronie",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "cronie-anacron-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "cronie-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "armv7hl"
					}
				],
				"i686": [
					{
						"name": "cronie-debugsource",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "cronie",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "cronie-noanacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "cronie-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "cronie-anacron-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "cronie-anacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "i686"
					}
				],
				"ppc64le": [
					{
						"name": "cronie-anacron-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "cronie-debugsource",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "cronie",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "cronie-noanacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "cronie-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "cronie-anacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "ppc64le"
					}
				],
				"s390x": [
					{
						"name": "cronie",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "cronie-anacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "cronie-noanacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "cronie-debugsource",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "cronie-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "cronie-anacron-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "s390x"
					}
				],
				"src": [
					{
						"name": "cronie",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "src"
					}
				],
				"x86_64": [
					{
						"name": "cronie",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "cronie-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "cronie-debugsource",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "cronie-anacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "cronie-anacron-debuginfo",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "cronie-noanacron",
						"version": "1.5.3",
						"release": "1.fc29",
						"arch": "x86_64"
					}
				]
			}
		}
	],
	"comments": [
		{
			"id": 908970,
			"text": "This update has been submitted for testing by tmraz. ",
			"timestamp": "2019-03-15 13:58:19",
			"update_id": 133860,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 909483,
			"text": "This update has been pushed to testing.",
			"timestamp": "2019-03-16 18:50:36",
			"update_id": 133860,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 909878,
			"karma": 1,
			"text": "works for me",
			"timestamp": "2019-03-17 20:18:26",
			"update_id": 133860,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/a3febfef42f58ed535f3c3a3cf9743653cd774dbb6e4554e2ce7c847d9802b6c?s=24&d=retro",
				"email": "cserpentis@gmail.com",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "trust admins"
					}
				],
				"id": 739,
				"name": "cserpentis",
				"openid": "cserpentis.id.fedoraproject.org"
			},
			"user_id": 739
		},
		{
			"id": 909882,
			"karma": 1,
			"text": "Works great!  LGTM!  =)",
			"timestamp": "2019-03-17 20:18:44",
			"update_id": 133860,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/28bacfa3be75032390ac2c7a34599aeb825c3b62b2c55ef64821d2fc023b56cb?s=24&d=retro",
				"email": "besser82.fpo@gmail.com",
				"groups": [
					{
						"name": "proventesters"
					},
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "ambassadors"
					},
					{
						"name": "designteam"
					},
					{
						"name": "shogun-ca"
					},
					{
						"name": "alt-gtk-de-sig"
					},
					{
						"name": "trust admins"
					}
				],
				"id": 206,
				"name": "besser82",
				"openid": "besser82.id.fedoraproject.org"
			},
			"user_id": 206
		},
		{
			"id": 909883,
			"text": "This update has been submitted for batched by bodhi. ",
			"timestamp": "2019-03-17 20:18:44",
			"update_id": 133860,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"bug_feedback": [
				{
					"bug": {
						"bug_id": 1687688,
						"parent": true,
						"security": true,
						"title": "CVE-2019-9704 vixie-cron: calloc return value resulting in remote dos"
					},
					"bug_id": 1687688,
					"comment_id": 909892
				},
				{
					"bug": {
						"bug_id": 1687694,
						"parent": true,
						"security": true,
						"title": "CVE-2019-9705 vixie-cron: dos(memory consumption) via a large crontab file"
					},
					"bug_id": 1687694,
					"comment_id": 909892
				}
			],
			"id": 909892,
			"testcase_feedback": [
				{
					"comment_id": 909892,
					"testcase": {
						"id": 371,
						"name": "QA:Testcase crontabs"
					},
					"testcase_id": 371
				}
			],
			"text": "Hmm, my cron jobs did not run after upgrade, so with this being pushed to stable, others may get stung too...",
			"timestamp": "2019-03-17 20:20:04",
			"update_id": 133860,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
				"email": "bojan@rexursive.com",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "trust admins"
					}
				],
				"id": 198,
				"name": "bojan",
				"openid": "bojan.id.fedoraproject.org"
			},
			"user_id": 198
		},
		{
			"bug_feedback": [
				{
					"bug": {
						"bug_id": 1687688,
						"parent": true,
						"security": true,
						"title": "CVE-2019-9704 vixie-cron: calloc return value resulting in remote dos"
					},
					"bug_id": 1687688,
					"comment_id": 909908
				},
				{
					"bug": {
						"bug_id": 1687694,
						"parent": true,
						"security": true,
						"title": "CVE-2019-9705 vixie-cron: dos(memory consumption) via a large crontab file"
					},
					"bug_id": 1687694,
					"comment_id": 909908
				}
			],
			"id": 909908,
			"karma": -1,
			"karma_critpath": -1,
			"testcase_feedback": [
				{
					"comment_id": 909908,
					"testcase": {
						"id": 371,
						"name": "QA:Testcase crontabs"
					},
					"testcase_id": 371
				}
			],
			"text": "I'm going to -1 this after checking the logs. User crontabs were ignored after upgrade, so maintainer should have a look before this hits stable.",
			"timestamp": "2019-03-17 20:52:20",
			"update_id": 133860,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/adb4c3f28bd6969871988f992b1bf71e404f03e3570fbfa0fc9bd7d0bf0003f8?s=24&d=retro",
				"email": "bojan@rexursive.com",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "trust admins"
					}
				],
				"id": 198,
				"name": "bojan",
				"openid": "bojan.id.fedoraproject.org"
			},
			"user_id": 198
		},
		{
			"id": 909909,
			"text": "Bodhi is disabling automatic push to stable due to negative karma. The maintainer may push manually if they determine that the issue is not severe.",
			"timestamp": "2019-03-17 20:52:20",
			"update_id": 133860,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 909996,
			"text": "Bodhi is unable to request this update for stabilization: This critical path update has not yet been approved for pushing to the stable repository.  It must first reach a karma of 2, consisting of 0 positive karma from proventesters, along with 2 additional karma from the community. Or, it must spend 14 days in testing without any negative feedback Additionally, it must pass automated tests.",
			"timestamp": "2019-03-17 23:46:25",
			"update_id": 133860,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 910190,
			"text": "This update has been unpushed.",
			"timestamp": "2019-03-18 10:00:12",
			"update_id": 133860,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/53b59528dfe2f17c94c39e505ceda177ade22bcd6a9cf96120f71e439ad70169?s=24&d=retro",
				"email": "tm@t8m.info",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "provenpackager"
					}
				],
				"id": 462,
				"name": "tmraz",
				"openid": "tmraz.id.fedoraproject.org"
			},
			"user_id": 462
		}
	],
	"content_type": "rpm",
	"critpath": true,
	"date_pushed": "2019-03-16 18:50:07",
	"date_submitted": "2019-03-15 13:58:19",
	"date_testing": "2019-03-16 18:50:07",
	"karma": 1,
	"meets_testing_requirements": true,
	"notes": "Minor security and bugfix update to version 1.5.3.\n",
	"release": {
		"branch": "f29",
		"candidate_tag": "f29-updates-candidate",
		"composed_by_bodhi": true,
		"create_automatic_updates": false,
		"dist_tag": "f29",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 29",
		"mail_template": "fedora_errata_template",
		"name": "F29",
		"override_tag": "f29-override",
		"package_manager": "dnf",
		"pending_signing_tag": "f29-signing-pending",
		"pending_stable_tag": "f29-updates-pending",
		"pending_testing_tag": "f29-updates-testing-pending",
		"stable_tag": "f29-updates",
		"state": "archived",
		"testing_repository": "updates-testing",
		"testing_tag": "f29-updates-testing",
		"version": "29"
	},
	"severity": "low",
	"stable_karma": 2,
	"status": "unpushed",
	"suggest": "unspecified",
	"test_cases": [
		{
			"id": 371,
			"name": "QA:Testcase crontabs"
		}
	],
	"test_gating_status": "ignored",
	"title": "cronie-1.5.3-1.fc29",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2019-768ed4fddc",
	"unstable_karma": -2,
	"updateid": "FEDORA-2019-768ed4fddc",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/53b59528dfe2f17c94c39e505ceda177ade22bcd6a9cf96120f71e439ad70169?s=24&d=retro",
		"email": "tm@t8m.info",
		"groups": [
			{
				"name": "packager"
			},
			{
				"name": "provenpackager"
			}
		],
		"id": 462,
		"name": "tmraz",
		"openid": "tmraz.id.fedoraproject.org"
	},
	"version_hash": "d9f000bc3490e462ddcfd2bc342bd643bf9febb3"
}
